DPG/Tasks/AOTTrack/qaMatchEff.cxx

```diff
@@ -1128,8 +1358,86 @@ struct qaMatchEff {
       //
       // all tracks, no conditions
       //
+
+      //  TPC clusters - all particles all dets
+      //
+      if constexpr (IS_MC) { ////////////////////////   MC
+        histos.get<TH1>(HIST("MC/TPCclust/tpcNClsFound"))->Fill(clustpc);
+        histos.get<TH1>(HIST("MC/TPCclust/tpcNClsFindable"))->Fill(findcltpc);
+        histos.get<TH1>(HIST("MC/TPCclust/tpcCrossedRows"))->Fill(crowstpc);
+        histos.get<TH1>(HIST("MC/TPCclust/tpcsFindableMinusCrossedRows"))->Fill(crowstpc);
+      } else {
+        histos.get<TH1>(HIST("data/TPCclust/tpcNClsFound"))->Fill(clustpc);
+        histos.get<TH1>(HIST("data/TPCclust/tpcNClsFindable"))->Fill(findcltpc);
+        histos.get<TH1>(HIST("data/TPCclust/tpcCrossedRows"))->Fill(crowstpc);
+        histos.get<TH1>(HIST("data/TPCclust/tpcsFindableMinusCrossedRows"))->Fill(finclusmincrotpc);
+      }
+
+      //  TPC clusters all pions
+      //
+      if (isPion) {
+        if constexpr (IS_MC) { ////////////////////////   MC
+          histos.get<TH1>(HIST("MC/TPCclust/tpcNClsFound_pi"))->Fill(clustpc);
+          histos.get<TH1>(HIST("MC/TPCclust/tpcNClsFindable_pi"))->Fill(findcltpc);
+          histos.get<TH1>(HIST("MC/TPCclust/tpcCrossedRows_pi"))->Fill(crowstpc);
+          histos.get<TH1>(HIST("MC/TPCclust/tpcsFindableMinusCrossedRows_pi"))->Fill(crowstpc);
```

Comment:
in my opinion these lines must be put inside the scope here: https://github.com/AliceO2Group/O2Physics/pull/4123/files#diff-a7ac2a00ff9015a809fe16e13dad8125e5e29054d61b13c605d1ef7dee975d96R1364-R1368

In the way you are writing now, everything is protected by `isPion`, which implies the application of a PID selection and this is not what you usually show

Reply:
Right, it could be useful to make a "pears to pears" comparison with MC, but it is also true that can keep the old way to compare...12 more hosts, again not a big deal :(

Reply:
Sorry, probably I'm just confused due to the too large amount of filled histograms (we really must rearrange the task in the near future).

 1. Here you are proposing a few histograms to check what happens in MC when looking at tracks reconstructed as in data, w/o specific track selections
 2. In this place instead https://github.com/AliceO2Group/O2Physics/pull/4123/files#diff-a7ac2a00ff9015a809fe16e13dad8125e5e29054d61b13c605d1ef7dee975d96R2287-R2290 you look only at true pions reconstructed at least with TPC
 
 Am I right? Then I would change the name of the histos in case 1. e.g. by adding a suffix (e.g. `_PID` or something that allows us to understand that it is as in data).